src/main/utils/appdir.js

```diff
@@ -0,0 +1,18 @@
+let appdir;
+
+if (process.env.APPDATA) {
```

Comment:
You need to have `const process = process.env` here like before or my environment variable transform breaks this code